### DISCUSSION TITLE
Does IINA have an EQ setting?

### DISCUSSION THREAD

**Discussion by birdjumper** [iina/iina#5118] [General]
Hi

I've seen references to it but I can't find anything in the menus. 

Does it have EQ settings? Did it used to?

Thanks

**Comment by low-batt** [comment]
It is on the quick settings panel. Under the `Audio` menu click on `Show Audio Panel`. The panel will appear on the right side of the window. Scroll down to the bottom and you will see the equalizer. There is also an equalizer on the video panel.

**Reply by birdjumper** [reply]
Thanks for the reply. Show Audio Panel is greyed out and I can't select it. Why is that?

My system is:

Mac M1 RAM 16GB running macOS Sonoma v14.5, Logic Pro 11.0.0, IINA v1.3.5 Build 141 mpv 0.35.0-419-gf79458476b-dirty FFmpeg 6.0

**Comment by birdjumper** [comment]
OK, after digging around, I understand I needed to exit Music Mode in order to access it. That was really hard to find!

I've been using IINA for a couple of years and I think that's the 1st time I've seen the EQ settings!

**Reply by low-batt** [reply]
Sorry about that. I failed to think about Music Mode. Glad you figured it out. Good point about it being hidden for users that mostly use Music Mode.
